windows-server-container-tools/Debug-ContainerHost/Debug-ContainerHost.ps1

```diff
@@ -88,6 +98,57 @@ Describe "The right container base images are installed" {
     }
 }
 
+Describe "Container network is created" {
+   $networksListOutput = docker.exe network ls
```

Comment:
I think we should avoid invoking `docker.exe network ls` twice if possible.

I recommend either:
- Move the first call from line 52 to here
- Parse the dockernetworks.txt here

I prefer the first option so that the path test case would pass if `docker.exe network ls` is not functional.